src/gp_instance.hpp

```diff
@@ -35,16 +58,70 @@ class GPInstance {
   // and PCSS bitsets are at the end.
   // The collection of rootsplits, with the same indexing as in the indexer_.
   BitsetVector rootsplits_;
-  // A map going from the index of a PCSS to its child.
+  // A map going from the index of a PCSP to its child.
   SizeBitsetMap index_to_child_;
   // A map going from a parent subsplit to the range of indices in
   // sbn_parameters_ with its children. See the definition of Range for the indexing
   // convention.
   BitsetSizePairMap parent_to_range_;
 
+  // The first entries are reserved for fake subsplits.
+  // We will adopt the convention of denoting a trivial subsplit by placing
+  // 0's in the first chunk.
+  // The next set of entries store the rootsplits.
+  // The remaining entries - 1 are for all other subsplits.
+  // The last entry is saved for pseudoroot -- for passing stationary.
+  // The pseudoroot is a Bitset with all 1's.
+  std::unordered_map<Bitset, size_t> subsplit_to_index_;
```

Comment:
There are different indices going on here, right? The PLVs have a different indexing system than the others? Would it be possible to explain that in the code?